src/NzbDrone.Core/MediaFiles/EpisodeImport/ImportApprovedEpisodes.cs

```diff
@@ -46,12 +46,13 @@ public class ImportApprovedEpisodes : IImportApprovedEpisodes
 
         public List<ImportResult> Import(List<ImportDecision> decisions, bool newDownload, DownloadClientItem downloadClientItem = null, ImportMode importMode = ImportMode.Auto)
         {
-            var qualifiedImports = decisions.Where(c => c.Approved)
-               .GroupBy(c => c.LocalEpisode.Series.Id, (i, s) => s
```

Comment:
`s` is IEnumerable and we enumerate it twice. I am not sure about backing type, but the warning is fixed and according to the benchmark the new version consumes a bit less memory and works a bit faster